src/AWSMetadataUtilities.jl

```diff
@@ -16,13 +17,43 @@ Get a list of all AWS service API definition files from the `awsk-sdk-js` GitHub
 - `Array{OrderedDict}`: Array of Dictionaries which contains information about each AWS Service
 """
 function _get_aws_sdk_js_files()
-    headers = ["User-Agent" => "JuliaCloud/AWS.jl"]
-    url = "https://api.github.com/repos/aws/aws-sdk-js/contents/apis"
+    aws_sdk_js_url = "https://api.github.com/repos/aws/aws-sdk-js"
+
+    function _request(url::String)
+        response = HTTP.get(url, headers)
+
+        return JSON.parse(String(response.body), dicttype=OrderedDict)
+    end
+
+    function _get_master_sha()
+        url_master = string(aws_sdk_js_url, "/commits/master")
+        resp = _request(url_master)
 
-    req = @mock HTTP.get(url, headers)
-    files = JSON.parse(String(req.body), dicttype=OrderedDict)
-    filter!(f -> endswith(f["name"], ".normal.json"), files)  # Only get ${Service}.normal.json files
-    files = _filter_latest_service_version(files)
+        return resp["sha"]
+    end
+
+    function _get_tree(master_sha::String)
+        url_tree = string(aws_sdk_js_url, "/git/trees/", master_sha)
+        resp = _request(url_tree)
+
+        return [t for t in resp["tree"] if t["path"] == "apis"][1]["url"]
+    end
+
+    function _get_api_files(url_apis::String)
+        resp = _request(url_apis)
+
+        if resp["truncated"]
+            throw(FileTruncated("There are more API files that GitHub can send."))
```

Comment:
Maybe this? I think it should be clearer than what's written but I'm not attached to this suggestion
```suggestion
            throw(FileTruncated("GitHub API file limit exceeded"))
```